app/Helpers/database/user-activity.php

```diff
@@ -1,110 +1,27 @@
 <?php
 
-use App\Community\Enums\ActivityType;
 use App\Community\Enums\ArticleType;
 use App\Community\Models\Comment;
-use App\Community\Models\UserActivityLegacy;
 use App\Platform\Enums\AchievementFlag;
 use App\Platform\Models\Game;
 use App\Platform\Models\PlayerAchievement;
 use App\Platform\Models\PlayerSession;
 use App\Site\Enums\Permissions;
 use App\Site\Models\User;
 use App\Support\Cache\CacheKey;
-use Carbon\Carbon;
 use Illuminate\Support\Facades\Cache;
 
 /**
  * @deprecated see WriteUserActivity listener
  */
-function postActivity(string|User $userIn, int $type, ?int $data = null, ?int $data2 = null): bool
+function updateActivity(int $activityID): void
```

Comment:
still needed?

Reply:
Nope. That was previously removed (https://github.com/RetroAchievements/RAWeb/pull/1923/files#diff-4eb28129e711e17b32a464138654fc22cd98c92b7c74d5413a64cea0786e79b8L37). I'm guessing it got resurrected by a merge conflict.